### ISSUE TITLE
Make side nav collapsible

### ISSUE THREAD

**Issue by edongashi** [diffix/desktop#145]

**Comment by edongashi** [comment]
> Great idea!

Is it still?

**Comment by cristianberneanu** [comment]
I don't think it is super useful, that's why I labeled it as "low-priority", but it would help a bit when working with many columns on a narrow screen.

**Comment by pdobacz** [comment]
![Screenshot from 2021-11-23 14-17-09](https://user-images.githubusercontent.com/5735525/143030953-e196b828-787c-41c9-8824-21a0412ba7e3.png)

I have time-boxed the effort to style the collapsible nav bar, and this is how far I got. I still have problems with the scrollbars and some other details, and a lot of cleanup. This is the version which uses Ant Design's `Layout`.

There also is a possibility of not using Ant Design's `Layout` and doing it manually, which isn't too bad or messy, just slightly less "slick". It would also require some additional styling efforts.

In terms of priority - I keep it as a back-burner task, I'm happy to continue in between other work. The question is - do we want this at all or not? I'm on the fence. I like the extra space reclaimed and less UI-noise, but the steps are non-standard enough to make the icons meaning a bit obscure anyway. A lot depends also on whether we see the documentation window as obligatory (to display disclaimers and educate) or optional (just to get the user started).

Then, as long as we're not remembering the UI setup between runs, having to collapse manually everytime might spoil this feature anyway.

**Comment by cristianberneanu** [comment]
I think it is fine to treat the sidebar as optional.

>  having to collapse manually everytime might spoil this feature anyway.

This part also seems fine to me.

**Comment by pdobacz** [comment]
After #269 this seems to bring too little benefit for the complexity added (and also might be confusing), so closing this as won't do, until we have more ideas/requirements, see [comment](https://github.com/diffix/desktop/pull/269#issuecomment-979143956)

**Comment by cristianberneanu** [comment]
Sounds fair. Thank you for experimenting on this!
